kalite/main/content_rating_models.py

```diff
@@ -5,14 +8,28 @@
 from securesync.models import DeferredCountSyncedModel
 
 class ContentRating(DeferredCountSyncedModel):
+
+    minversion = "0.15.0"
+
     class Meta:
         unique_together = ("content_kind", "content_id", "user")

```

Comment:
Just to avoid weird possible edge-case collisions, probably best to update this to include `content_source` as well. May need to then regenerate the migration.